src/test/java/org/broadinstitute/hellbender/utils/python/PythonScriptExecutorUnitTest.java

```diff
@@ -16,23 +16,23 @@
 public class PythonScriptExecutorUnitTest extends GATKBaseTest {
     final String HELLO_WORLD_SCRIPT = "print (\"hello, world\")";
 
-    @Test(groups = {"python"})
```

Comment:
Good point. There is still a "python" group (though it probably should be called "conda" because its used for things that depend on the conda env). There is also a travis matrix entry for it, and tests in the "python" group only run in that one travis job. All of the VQSR-CNN and gCNV tests will go in that group soon as well .

There isn't anything about these tests that requires conda, but I could go either way. Maybe that does make the most sense though. If I leave these in the python group, then I would probably also put all of the StreamingPythonExecutor tests in that group as well. Let me know what you think.

Reply:
Going put all of these in the python group.